vaadinfrontend/src/main/java/life/qbic/datamanager/views/project/create/ProjectInformationDialog.java

```diff
@@ -123,23 +123,15 @@ public String getObjective() {
     return projectObjective.getValue();
   }
 
-  public String getExperimentalDesign() {
-    return experimentalDesignField.getValue();
+  public String getExperimentalDesign() { return experimentalDesignField.getValue(); }
```

Comment:
What happened here?
```suggestion
public String getExperimentalDesign() {
 return experimentalDesignField.getValue(); 
}
```
Please do not remove the new-line characters. Your IDE will automatically display it in one line but our code style expects the body in a new line.